### PR TITLE
Chore: Migrate some more deprecated SCSS

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- converts places using `pull-right`/`pull-left` classes
  - most places they weren't necessary at all (e.g. used in a table)
  - adds them to angular file as there's some angular partials still using them
  - there is 1 external plugin using `pull-left` that will need an issue raising against
- removes some other font awesome scss as it's not being used at all 
- migrates some emotion to object syntax in files i was touching already

**Why do we need this feature?**

- clean up some old deprecated SCSS

**Who is this feature for?**

- grafana devs

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/features/storage/RootView.tsx [hunk 1]
```diff
@@ -83,9 +83,8 @@ export function RootView({ root, onPathChange }: Props) {
         <InlineField grow>
           <FilterInput placeholder="Search Storage" value={searchQuery} onChange={setSearchQuery} />
         </InlineField>
-        <Button className="pull-right" onClick={() => onPathChange('', StorageView.AddRoot)}>
-          Add Root
-        </Button>
+        <div className="page-action-bar__spacer" />
```
Comment (Clarity-89) [inline public/app/features/storage/RootView.tsx:86]: Would the `Space` component work here?

Comment (ashharrison90) [reply public/app/features/storage/RootView.tsx:86]: don't think so cause it needs to flex. looks like `Space` only supports hardcoded values right now 🤔

Comment (Clarity-89) [reply public/app/features/storage/RootView.tsx:86]: I think an empty Stack component could work then? If we want to get rid of this class as well.

Comment (ashharrison90) [reply public/app/features/storage/RootView.tsx:86]: in this case it's already a global emotion style, so i'm not too fussed about getting rid of it immediately 🤷

**Review by Clarity-89 — APPROVED** [review]

**Pull request merged**